python-libraries/data-platform-catalogue/tests/snapshots/datahub_create_athena_table.json

```diff
@@ -7,18 +20,14 @@
     "aspect": {
         "json": {
             "customProperties": {
-                "owner_display_name": "April Gonzalez",
-                "maintainer": "j.shelvey@digital.justice.gov.uk",
-                "maintainer_display_name": "Jonjo Shelvey",
-                "email": "justice@justice.gov.uk",
-                "retention_period_in_days": "365",
-                "subdomain": "Legal Aid",
+                "status": "DEV",
                 "dpia_required": "False",
                 "dpia_location": "None",
-                "last_updated": "2020-05-17 00:00:00",
-                "creation_date": "2020-05-17 00:00:00",
+                "data_sensitivity_level": "OFFICIAL",
+                "where_to_access_dataset": "analytical_platform",
+                "source_dataset_name": "",
                 "s3_location": "s3://databucket/",
-                "status": "PROD"
+                "row_count": "None"
```

Comment:
This seems like a bug - I would expect the row_count property to default to an empty string if not defined